components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1154,58 +1156,86 @@ void AdsServiceImpl::URLRequest(
       const std::string& content_type,
       ads::URLRequestMethod method,
       ads::URLRequestCallback callback) {
-  net::URLFetcher::RequestType request_type = URLMethodToRequestType(method);
-
-  net::URLFetcher* fetcher = net::URLFetcher::Create(
-      GURL(url), request_type, this).release();
-  fetcher->SetRequestContext(g_browser_process->system_request_context());
-  fetcher->SetAutomaticallyRetryOnNetworkChanges(kRetriesCountOnNetworkChange);
-
+  net::NetworkTrafficAnnotationTag traffic_annotation =
```

Comment:
@bridiver  Please fill this annotation. I don't know any background for this service.

Reply:
cc @NejcZdovc